pkg/util/mount/mount_windows.go

```diff
@@ -0,0 +1,131 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+type Mounter struct {
+	mounterPath string
+}
+
+func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
+	if source == "tmpfs" || len(target) < 3 {
+		glog.Infof("azureMount: Skip mounting source (%q), target (%q)\n, with arguments (%q) in windows", source, target, options)
+		return nil
+	}
+
+	if !strings.HasPrefix(target, "c:") && !strings.HasPrefix(target, "C:") {
+		target = "c:" + target
+	}
+	parentDir := filepath.Dir(target)
+	err := os.MkdirAll(parentDir, 0755)
+	if err != nil {
+		glog.Infof("mkdir(%q) failed: %v\n", parentDir, err)
+		return err
+	}
+
+	if len(options) != 1 {
+		glog.Warningf("azureMount: mount options(%q) command(%n) not equal to 1, skip mounting.\n", options, len(options))
+		return nil
+	}
+	cmd := options[0]
+
+	driverLetter, err := getAvailabeDriverLetter()
+	if err != nil {
+		return err
+	}
+	driverPath := driverLetter + ":"
+	cmd += fmt.Sprintf(";New-SmbGlobalMapping -LocalPath %s -RemotePath %s -Credential $Credential", driverPath, source)
+
+	_, err = exec.Command("powershell", "/c", cmd).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("azureMount: SmbGlobalMapping failed: %v", err)
+	}
+
+	_, err = exec.Command("cmd", "/c", "mklink", "/D", target, driverPath).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("mklink failed: %v", err)
+	}
+
+	return nil
+}
+
+func (mounter *Mounter) Unmount(target string) error {
+	glog.Infof("azureMount: Unmount target (%q)\n", target)
+	output, err := exec.Command("cmd", "/c", "rmdir", target).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("Unmount failed: %v", err)
+	}
+	glog.Infof("azureMount: Unmount succeeded, output: %q", output)
+	return nil
+}
+
+func (mounter *Mounter) List() ([]MountPoint, error) {
+	return []MountPoint{}, nil
+}
+
+func (mounter *Mounter) IsMountPointMatch(mp MountPoint, dir string) bool {
+	return (mp.Path == dir)
+}
+
+func (mounter *Mounter) IsNotMountPoint(dir string) (bool, error) {
+	return IsNotMountPoint(mounter, dir)
+}
+
+func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
+	return true, nil
+}
+
+func (mounter *Mounter) GetDeviceNameFromMount(mountPath, pluginDir string) (string, error) {
+	return getDeviceNameFromMount(mounter, mountPath, pluginDir)
+}
+
+func (mounter *Mounter) DeviceOpened(pathname string) (bool, error) {
+	return false, nil
+}
+
+func (mounter *Mounter) PathIsDevice(pathname string) (bool, error) {
+	return true, nil
+}
+
+func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
+	return nil
+}
+
+func (mounter *SafeFormatAndMount) diskLooksUnformatted(disk string) (bool, error) {
```

Comment:
not used anywhere.

Used to be linux only, and has renamed, should remove.
https://github.com/kubernetes/kubernetes/pull/16948
https://github.com/kubernetes/kubernetes/pull/31515